src/kudu/consensus/raft_consensus.cc

```diff
@@ -2868,6 +2898,7 @@ void RaftConsensus::CompleteConfigChangeRoundUnlocked(ConsensusRound* round, con
       LOG_WITH_PREFIX_UNLOCKED(INFO) << "Aborting config change with OpId "
                                      << op_id << ": " << status.ToString();
       cmeta_->clear_pending_config();
+      CHECK_OK(routing_table_->UpdateRaftConfig(cmeta_->ActiveConfig()));
```

Comment:
we need to update the routing topology as soon as we have a pending config. Right?
Not just at commit time.